src/assets/styles/styles/custom/_button.scss

```diff
@@ -52,3 +52,29 @@
   display: flex;
   justify-content: center;
 }
+
+.btn-dropdown {
+  border-radius: 0px;
+  &:hover {
+    cursor: pointer;
+    font-weight: 600;
+    box-shadow: 0px 0px;
+  }
+  float: left;
+  clear: left;
+  padding: 5px 7px 5px;
+  line-height: 20px;
+}
```

Comment:
Moim zdaniem, lepiej jest przenieść to bezpośrednio do klasy `.dropdown-buttons-container`. To ma taką zaletę, że te style są przeznaczone tylko do dropdowna na co wskazuje nazwa `btn-dropdown`, więc było by fajnie zrobić tak, żeby nie można było tego użyć w jakimś innym miejscu, żeby ktoś przez przypadek tego nie użył. Chyba że uważasz, że można by użyć je też w innych miejscach. To w takim razie, proponowałbym zmienić nazwę tej klasy na coś bardziej generyczne ;) 

PS.
To o czym mówię wyglądało by jakoś tak:

```scss
.dropdown-buttons-container {
  overflow: hidden;
  border-bottom-left-radius: $btn-border-radius;
  border-bottom-right-radius: $btn-border-radius;
  border-top-left-radius: 0px;
  border-top-right-radius: 0px;
  box-shadow: 0 2.8px 2.2px $gray-400, 0 6.7px 5.3px $gray-400;
  position: relative;
  top: -20px;
  padding-bottom: 10px;
  padding-top: 25px;
  
  button {
    border-radius: 0px;
    &:hover {
      cursor: pointer;
      font-weight: 600;
      box-shadow: 0px 0px;
    }
    float: left;
    clear: left;
    padding: 5px 7px 5px;
    line-height: 20px;
 }
}
```
Zauważ, ze podalem `button`, a nie `btn-dropdown`. Zrobiłem tak, bo tego całkiem wystarczy - css nadpisze wszystkie style co były używane wcześniej i użyje tych co są tutaj.

Reply:
poprawiłam :)